### DISCUSSION TITLE
Fix biases in adni clinical scores

### DISCUSSION THREAD

**Discussion by omar-rifai** [aramis-lab/clinica#436] [Q&A]
Based on [work](https://groups.google.com/g/adni-data/c/hfTaDDpBnDY) done by @emaheux, some clinical scores in clinica need revising. In particular for ADAS-Cog13 the following issues have been found:

- Q4 (delayed word recall): in ADNI3, the value "10" in Q4TASK is not interpreted as worst possible score (as specified in dictionary and performed in previous phases) but surprisingly interpreted instead as 2 good answers (furthermore in a few rows for Q4TASK and a few others in Q1TR* this "10" special value is used as the index of a correctly recalled word index - supposed to be in 0..9 as specified in dictionary). Fixing this (~150 rows) solved Q4 distribution bias for ADNI3.
- Q8 (word recognition): in ADNI3, as specified in manual "ADAS Word List 1 will be used at each annual In-Clinic Visit", however it seems to me that the scoring rule from recorded CRF sub-items was not updated accordingly, continuing to score Q8WORD* with respect to the ADNI GO/2 word lists cycle. Fixing this (~1750 rows) solved Q8 distribution bias for ADNI3.
- Q13 (number cancellation task): in ADNI3, it seems to me that the 60s scoring thresholds were used for the number cancellation task, unlike in earlier phases where the 45s thresholds were used and despite no information of a protocol change on this ADAS item. Using again the 45s scoring thresholds solved Q13 distribution bias for ADNI3 (~1450 rows).

**Comment by emaheux** [comment]
These issues have been reported to the ADNI Clinical Core team (cf. https://groups.google.com/g/adni-data/c/hfTaDDpBnDY), which has confirmed the issues on Q4 & Q8 (for Q13 they didn't mention). They may have fixed those bugs but I haven't checked so far!

**Reply by ghisvail** [reply]
It does not look like there has been an update yet, since there is no `Version: $DATE` besides the link to the ADAS files. No news from them for more than 3 months, maybe it's worth bumping the thread?

**Reply by emaheux** [reply]
There are no such versions dates for neuropsychological assessments tables because most of them are updated on a daily basis!

Good news: I just quickly double-checked and they effectively implement the changes :) on Q4, Q8 and Q13 (although not reported in their reply from June). They operated the change the 2021-06-17 apparently (cf. update_stamp, last column of data) .

**Reply by ghisvail** [reply]
> most of them are updated on a daily basis!

Waoh, that's awesome!

> They operated the change the 2021-06-17 apparently (cf. update_stamp, last column of data)

Where do you see these timestamps?

**Reply by emaheux** [reply]
This is the last column of most of their data tables (csv files), for instance here for `ADAS_ADNIGO23.csv`
